config/features.yml

```diff
@@ -2354,7 +2362,7 @@ features:
     actor_type: user
     description: Enables redaction for the Accredited Representative Portal POA requests
   calculator_constants_versioning:
-    actor_type: 
+    actor_type:
```

Comment:
Please remove actor_type if not defining it.

Reply:
@RachalCassity I didn't actually add this key, Rubocop removed the blank space from the end of the line